src/components/icons/icon.spec.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Icon, {types, colors} from './Icon';
+import {shallow} from 'enzyme';
+
+test('render', () => {
+  const icon = shallow(
+    <Icon type={types.answer}/>
+  );
+
+  expect(icon.hasClass('sg-icon')).toEqual(true);
+  expect(icon.find('use')).toHaveLength(1);
+});
+
+test('error when no type', () => {
+  const spy = jest.spyOn(console, 'error');
```

Comment:
jest contains few libraries for testing and you don't need define everything. Variables test, expect, describe is defined by jest globally. For testing React we need only import enzyme library with one allow fast rendering components for tests